client/app/cloud/project/storage/storage.html

```diff
@@ -1,62 +1,61 @@
-<div class="cui-page">
-    <div data-ng-include="'app/cloud/project/storage/storage-header.html'"></div>
-
-    <section class="cui-page__content">
-        <cui-page-content-title data-text="'storage_title' | translate">
-            <oui-spinner data-ng-show="loaders.storages"></oui-spinner>
-        </cui-page-content-title>
-
-        <div data-ng-if="storages !== null">
-            <a class="oui-button oui-button_secondary"
-               data-ui-sref="iaas.pci-project.storage.add-container"
-               write-right-required>
-                {{ 'storage_add_button'|translate }}
-            </a>
-            <div class="pull-right" data-ng-if="storages.length > 0">
-                <input type="search"
-                class="oui-input"
-                data-ng-model="filter.name"
-                data-ng-model-options="{ debounce: 200 }"
-                data-translate-attr="{ placeholder: 'storage_search' }"/>
-            </div>
+<section class="cui-page__content">
+
+    <cui-message-container data-messages="messages"></cui-message-container>
+
+    <cui-page-content-title data-text="'storage_title' | translate">
+        <oui-spinner data-ng-show="loaders.storages"></oui-spinner>
+    </cui-page-content-title>
+
+    <div data-ng-if="storages !== null">
+        <a class="oui-button oui-button_secondary"
+           data-ui-sref="iaas.pci-project.compute.storage.add-container"
+           write-right-required>
+            {{ 'storage_add_button'|translate }}
+        </a>
+        <div class="pull-right" data-ng-if="storages.length > 0">
```

Comment:
If you use `.pull-right` class, you should have a `.clearfix` on your the parent element, to avoid overlap